rules/style.js

```diff
@@ -121,7 +121,11 @@ module.exports = {
     "operator-linebreak": [2, "after"],
     // Enforce padding within blocks
     // http://eslint.org/docs/rules/padded-blocks
-    "padded-blocks": [2, "never"],
+    "padded-blocks": [2, {
+      "blocks": "never",
+      "classes": "always",
```

Comment:
I don't have strong opinions on padding. It's nice to have in function blocks sometimes to improve readability.